ddocs/medic-client/views/reports_by_form_year_month_places/map.js

```diff
@@ -1,71 +1,24 @@
 // data record must adhere to property name of `month`
 function(doc) {
-  if (doc.type !== 'data_record') {
-    return;
+  if (doc.type === 'data_record' &&
+      doc.form &&
+      doc.fields &&
+      doc.fields.year &&
+      doc.fields.month) {
+    var year = parseInt(doc.fields.year, 10);
+    var month = parseInt(doc.fields.month, 10);
+    var clinic = doc.contact && doc.contact.parent;
+    var healthCenter = clinic && clinic.parent;
+    var district = healthCenter && healthCenter.parent;
+    emit(
+      [ doc.form, year, month ],
+      {
+        isValid: (doc.errors && doc.errors.length === 0),
+        clinicId: clinic && clinic._id,
+        healthCenterId: healthCenter && healthCenter._id,
+        districtId: district && district._id,
+        month: month
```

Comment:
We can't get the reporting name or phone because it's on a linked doc. So instead we return the ids and go from there.